app/search_builders/report_search_builder.rb

```diff
@@ -8,5 +8,9 @@ class ReportSearchBuilder < Blacklight::SearchBuilder
 
   self.default_processor_chain -= [:add_facetting_to_solr] # remove faceting from reports
 
-  self.default_processor_chain += [:add_date_field_queries] # ensure date field queries work
+  self.default_processor_chain += [
+    :add_access_controls_to_solr_params, # enforce restrictions
+    :add_date_field_queries,   # ensure date field queries work
```

Comment:
kill the comma, methinks

Reply:
yup, and a blank line -- fixed, rebased, pushed